bootstrapping_tools/bootstrapping.py

```diff
@@ -34,6 +65,15 @@ def remove_distribution_outliers(data, number_of_std=2.698):
 
 
 def tukey_fences(data, fence_width=1.5):
+    """This function filters an array using Tukey fences method
```

Comment:
Creo que el _"This function"_ sale sobrando:

```suggestion
    """Filter an array using Tukey fences method
```